_includes/header.html

```diff
@@ -32,10 +32,10 @@
           {% endif %}
         </a>
         <div
-          class="syn-page-header-divider vertical syn-ml-16 syn-mr-16 {% if site.enterprise %}syn-hidden-lg syn-hidden-md syn-hidden-sm{% endif %}"
+          class="syn-page-header-divider vertical syn-ml-16 syn-mr-16 syn-hidden-lg syn-hidden-md syn-hidden-sm"
```

Comment:
This doesn't seem like it should fix, it just ought to make the brand title hidden all the time?

Reply:
These are just the initial styles that happen on page load. [This script ](https://github.com/algorithmiaio/dev-center/blob/f9e1449d61b0b74a90084b5cf7219c6780e93d37/_layouts/default.html#L82)is run for both enterprise and public marketplace, and the `.hidden` classes are removed as soon as the API config values are fetched [here](https://github.com/algorithmiaio/dev-center/blob/develop/js/stageCustomization.js#L22).

Reply:
Ah, okay! Thanks for clarifying!